R/dots.R

```diff
@@ -97,7 +97,8 @@ dots_clean_empty <- function(dots, is_empty, ignore_empty) {
 
   if (n_dots) {
     which <- match.arg(ignore_empty, c("trailing", "none", "all"))
-    switch(which,
+    switch(
+      which,
```

Comment:
I thought the `switch` and the arguments are part of the same nest (=sub-tibble of the nested parse data)?

I'm fine with

```r
switch(cond,
  x, 
  y
)
ifelse(cond,
  true_part,
  false_part
)
```

A simpler solution that seems enough for now: The line break after the opening paren is added only if `strict = TRUE`. I'm not sure what the current setup is.